13output.tex

```diff
@@ -480,9 +480,9 @@
 	\begin{itemize}
 		\item This section contains the selectivities organized as a database, rather than as a set of vectors.
 	\end{itemize}
-	\item SPR/YPR Profile
-	\item Global MSY Report
-	\item Dynamic Bzero Report
+	\item \gls{spr}/\gls{ypr} Profile
+	\item Global \gls{msy} Report
+	\item Dynamic Bzero (B0) Report
```

Comment:
Why not $B_{0}$?

Reply:
I would also suggest replacing Bzero with Unfished Biomass and then have the abbreviation in the parathesis.

Reply:
I see in other places B0 is called Virgin Spawning Biomass. Should it all be Unfished Biomass or Virgin Spawning Biomass (lines 78 and 80 of 13output.tex, also lines 217 in 6starter.tex)?

Reply:
Since the input and output text files can't have subscripts but the user manual can, I like the idea of replacing "Dynamic Bzero" with "Dynamic _B_<sub>0</sub>" in the manual.

In SS3, I think "Virgin Spawning Biomass" and B0 are interchangeable, but I think "Unfished Biomass" refers to the spawning biomass after application of any equilibrium F and may therefore differ from B0.

Reply:
I like use of subscript.
in ADMB, log() is used to implement ln(), e.g. use of base e.  So I commonly write log() when I mean ln().  Not sure what to recommend for the manual; perhaps a glossary entry.
Virgin spawning biomass starts the time series and commonly serves as basis for depletion calculations.  It uses start year biology in its calculation.  
Unfished spawning biomass is used in the benchmark calculations (e.g. MSY and SPR).  It uses biology from the years selected as the Bmark_yrs.  So, it can differ from the virgin spawning biomass if there is time-varying biology.

Reply:
I've been in the habit of always using log() to mean ln(), but reading more about the two options, it's clear that the interpretation of log() depends on the discipline, context, or country (see https://en.wikipedia.org/wiki/Natural_logarithm#Notational_conventions for instance).

Since SS3 is used by folks with different backgrounds I'm now convinced that using ln() throughout the documentation (and my own future writing) makes sense for clarity. I would still use the spelling "log" in terms like log-likelihood, and I'm not worried about how it's represented in the code, or the output files, but in user manual equations I like the idea of switching from log() to ln().

Reply:
Okay, converting the log -> ln to a new issue and will do in a separate PR since there are already a ton of changes in this one.